libs/core/src/lib/multi-input/multi-input.component.html

```diff
@@ -100,7 +103,7 @@
         <li
             *ngFor="let value of displayedValues"
             fd-list-item
-            (click)="openChangeHandle(false)"
+            (click)="!mobile ? openChangeHandle(false): openChangeHandle(true)"
```

Comment:
Can be shortened into `openChangeHandle(!!mobile)`.

Reply:
done